DomainServices/Services/CustomerService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using DomainModels.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DomainServices.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private List<Customer> _customerList = new();
+
+        public long Create(Customer customer)
+        {
+            var cpfAlreadyExists = _customerList.Any(customerElement => customerElement.Cpf == customer.Cpf);
+
+            if (cpfAlreadyExists) throw new ArgumentException($"Customer for Cpf: {customer.Cpf} already exists!");
```

Comment:
Acredito que essa variável é desnecessária, pois ele é usada somente uma vez nesse método, podes colocar direto no if.
```suggestion
            if (_customerList.Any(customerElement => customerElement.Cpf == customer.Cpf)) throw new ArgumentException($"Customer for Cpf: {customer.Cpf} already exists!");
```